### ISSUE TITLE
Final discussion on package importer

### ISSUE THREAD

**Issue by adriatic** [monterey-framework/monterey#409]
### Intention

**I'm submitting a Discussion request**  

This document is created the collect the comments to the **[aurelia import](https://aurelia-ui-toolkits.gitbooks.io/monterey-developer-manual/content/infrastructure/aurelia_import/design.html)** component of the aurelia-cli tool.

**Comment by adriatic** [comment]
I would propose to include the brief section under the name **Definition of terms** before any existing text to ensure that the words `import`, `install` and `configure` are understood as intended.

The term **`install`** refers to the process of fetching a package from a remote registry (`npm`, `jspm`) and copying it to the local storage (`node_modules`, `jspm_packages`, etc.)

**Comment by adriatic** [comment]
> When a strategy is unable to "handle" the configuration of the plugin,  the next strategy is ~~asked~~ applied.

**Comment by adriatic** [comment]
The word `determinate` (appearing twice) should be spelled as `determine`

![image](https://cloud.githubusercontent.com/assets/2712405/20545711/9bccd932-b0de-11e6-9dd0-1580fec8cee2.png)

**Comment by JeroenVinke** [comment]
Thanks, good feedback. 

> When a strategy is unable to "handle" the configuration of the plugin, the next strategy is applied.

Using "apply" here seems to imply that the next strategy cannot refuse the configuration and pass the request to its successor. Or am I misreading this?

**Comment by adriatic** [comment]
You are right - it's just the word "asked" is a bit strange at that place. So, here is what I propose

Then the importer will run through a list of (currently) 7 importer strategies. When a strategy is unable to "handle" the configuration of the plugin, it proceeds to the next strategy on the list.

**Comment by JeroenVinke** [comment]
Perfect!
